pkg/engine/lifecycletest/pulumi_test.go

```diff
@@ -127,6 +128,14 @@ func pickURN(t *testing.T, urns []resource.URN, names []string, target string) r
 
 func TestMain(m *testing.M) {
 	grpcDefault := flag.Bool("grpc-plugins", false, "enable or disable gRPC providers by default")
+	if runtime.GOOS == "windows" || runtime.GOOS == "darwin" {
```

Comment:
I just hit this when trying to run lifecycle tests locally on macOS (tests no longer run). Is #19675 imminent for macOS? If not, mind if I open a PR that allows the tests to run when an envvar is set?